### ISSUE TITLE
fscrypt is no longer experimental

### ISSUE THREAD

**Issue by nh2** [NixOS/nixpkgs#221731]
The nixpkgs fscrypt package is called `fscrypt-experimental`:

https://github.com/NixOS/nixpkgs/blob/2037f1bbe8cfea60fddd5e369c2bdceaf3abe4e2/pkgs/top-level/all-packages.nix#L12027-L12028

Since [this commit](https://github.com/google/fscrypt/commit/3f865a76c5891b1664136decb4f2eb6ee5391f57) upstream says:

>  README: remove note about stability
> A lot of people are already using fscrypt, so in practice we haven't been breaking backwards compatibility and aren't going to

So I think it's time to call it `fscrypt` in NixOS too, like most other distros do

---

CC maintainer @primeos

**Comment by primeos** [comment]
Yeah, +1, sounds good to me (it actually feels quite overdue). IIRC I added it as a precaution when it was still new so that no one would loose their data due to wrong expectations but I'm not sure if the `-experimental` suffix was ever a good idea tbh. Anyway, I've never heard of any major issues and users of encryption software should be aware that it's a good idea to have backups of at least the keys/headers/metadata when storing important data.

Will you send a PR or should I take care of it?

PS: If you're interested in maintaining fscrypt (and fscryptctl) that'd be welcome as well since I don't really use it much anymore.

**Comment by risicle** [comment]
Time's running out if you want this to happen before 23.11
